.github/scripts/install.sh

```diff
@@ -49,16 +50,21 @@ else
     # Remove packages we have subrepos for
     pip uninstall spyder-kernels -q -y
     pip uninstall python-lsp-server -q -y
-
+    pip uninstall qdarkstyle -q -y
 fi
 
-# This is necessary only for Windows (don't know why).
-if [ "$OS" = "win" ]; then
-    # Install python-lsp-server from our subrepo
-    pushd external-deps/python-lsp-server
+# Install subrepos in development mode
+for dep in $(ls external-deps)
+do
+    pushd external-deps/$dep
     pip install --no-deps -q -e .
     popd
-fi
+done
+
+# Install Spyder to test it as if it was properly installed
+pip uninstall spyder -q -y
+python setup.py bdist_wheel
+pip install --no-deps dist/spyder*.whl
```

Comment:
@ccordoba12, does it need to be installed? Is it not sufficient to do the following?
```suggestion
python setup.py egg_info
```

Reply:
You're right, that's enough.

Reply:
Actually, it works for me locally but it fails on CI (don't why). So I had to revert this change.